src/java/com/sarality/app/common/datetime/DateTimeBuilder.java

```diff
@@ -62,10 +62,22 @@ public static DateTimeBuilder time(int hour, int minute, int sec) {
   public static DateTimeBuilder date(int year, int month, int day) {
     return new DateTimeBuilder(DateTime.forDateOnly(year, month, day));
   }
-  
+
+  /**
+   * Returns DateTimeBuilder with the date set as the given year, month and day.
+   *
+   * @param year Year value of the date.
+   * @param month Month value of the date.
+   * @param day Day value of the date.
+   * @return A DateTimeBuilder with the date set as the given year, month and day.
+   */
+  public static DateTimeBuilder date(int year, int month, int day, int hour, int minutes, int seconds) {
```

Comment:
Why would you not call the DateTime class directly rather than do DateTimeBuilder.date(...).build()?

And if you include this here it should be called dateTime rather than date.